.changeset/short-eggs-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/react": patch
+---
+
+url encode file names
```

Comment:
Can you clarify this a bit? Hard to tell if we're talking about File inputs or some routing issue here

Reply:
Done